core/src/main/java/org/elasticsearch/action/ingest/SimulatePipelineRequest.java

```diff
@@ -41,7 +42,7 @@
 
 import static org.elasticsearch.ingest.IngestDocument.MetaData;
 
-public class SimulatePipelineRequest extends ActionRequest {
+public class SimulatePipelineRequest extends ActionRequest implements Writeable {
 
     private String id;
```

Comment:
Even if the transport-client request builder pattern is to be deprecated, I think it may be best for that to be done in its own context in separate PRs